build_bundle_image.sh

```diff
@@ -195,6 +195,13 @@ function get_latest_tomcat_version {
 
 	if [ -z "${LIFERAY_RELEASE_TEST_MODE}" ]
 	then
+		latest_tomcat_version=$(curl --silent "https://downloads.apache.org/tomcat/tomcat-9/" | grep --only-matching --perl-regexp "9\.\d+\.\d+" | sort --version-sort | tail --lines=1)
```

Comment:
@kiwm, please source format based on the existing `curls` examples.

In addition, please set this `curl` result to the variable `latest_tomcat_version` and refactor the tests to make them pass.

Lastly, please create a task ticket for we update this code to support tomcat 10.x before the release of 2025.q3.x.